src/SamplesApp/UITests.Shared/Helpers/SkiaSamplesAppHelper.cs

```diff
@@ -34,9 +32,9 @@ public static async Task SaveFile(string filePath, string content, CancellationT
 				var json = JsonSerializer.Serialize(new { FilePath = filePath, Content = content });
 				using (var client = new HttpClient())
 				{
-					var protocol = Eval("window.location.protocol");
-					var hostname = Eval("window.location.hostname");
-					if (!int.TryParse(Eval("window.location.port"), out var port))
+					var protocol = WebAssemblyImports.EvalString("window.location.protocol");
+					var hostname = WebAssemblyImports.EvalString("window.location.hostname");
+					if (!int.TryParse(WebAssemblyImports.EvalString("window.location.port"), out var port))
```

Comment:
Let's avoid doing evals this way. If we need those three, let's expose them as actual imports